index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Home</title>
+  <link href="./styles/normalize.css" rel="stylesheet">
+  <link href="./styles/icono.min.css" rel="stylesheet" >
+  <link href="./styles/main.css" rel="stylesheet">
+  <script src="https://use.fontawesome.com/ed5b29bc68.js"></script>
+</head>
+
+<body>
+  <header>
+    <li><a href="index.html"><i class="icono-home"></i></a></li>
+    <nav>
+      <li><a href="portfolio.html"> My Pieces </a></li>
+      <li><a href="code-journal.html"> Learning </a></li>
+      <li><a href="about.html"> Resume </a></li>
+    </nav>
+  </header>
+  <section id="index-section">
+    <div>
+      <h2> My Journey to Become a Software Developer </h2>
+    </div>
+    <div class="name">
+      <ul>
+        <li class="star">★</li>
+        <li><h1 id="name-h1"> Sofia Kim </h1></li>
+        <li class="star">★</li>
+      </ul>
+    </div>
+    <div id="p">
+      <p>
+        Former investment banker, aspiring finance professional including a successful tenure in one of the largest global investment banks as part of the Financial Control department, responsible for Product Control. Made the decision to transition from Finance to programming and am currently enrolled in Cohort 7 of Ada Developer’s Academy (Ada). Will be seeking fulltime internship from August 7, 2017 until the end of 2017. Expected graduation is in January of 2018, from which I will be seeking fulltime employment as a software developer.
+      </p>
+    </div>
+    <footer>
+      <h3>Let's Be Touched!!</h3>
+      <div class="icons">
+        <li><a href="https://www.linkedin.com/in/sofia-kim-20572792/"><i class="fa fa-linkedin" aria-hidden="true"></i> Linkedin </a></li>
+        <li><a href="https://github.com/Sofia15"><i class="fa fa-github-alt" aria-hidden="true"></i> Github </a></li>
+        <li><a href="mailto:sofiamhkim@gmail.com" ><i class="fa fa-envelope" aria-hidden="true" ></i>&nbsp;E-mail</a></li>
+      </div>
+    </footer>
+    <br />
+    <br />
+    <br />
+    <div class="images">
```

Comment:
I think I would argue that these images and the following quote are part of the footer.